File: C++/C++Basic/04.Conditional-Statements-Exercise/06.worldSwimmingRecord.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
    double worldRecord, distance, time;
    cin >> worldRecord >> distance >> time;

    double totalTime = distance * time + floor(distance / 15) * 12.5;
    
    cout.setf(ios::fixed);
    cout.precision(2);

    if (totalTime < worldRecord)
    {
        cout << "Yes, he succeeded! The new world record is " << totalTime << " seconds." << endl;
    }
    else
    {
       cout << "No, he failed! He was " << abs(totalTime - worldRecord) << " seconds slower." << endl;
    }

    return 0;
}

File: Python/Basic/04.Conditional-Statements-Exercise/06.world_swimming_record.py
import math

word_records = float(input())
distance = float(input())
time_per_meter= float(input())

time_distance = distance * time_per_meter
delay_time = math.floor(distance / 15) * 12.5

total_time = time_distance + delay_time

if total_time < word_records:
    print(f" Yes, he succeeded! The new world record is {total_time:.2f} seconds.")
else:
    print(f"No, he failed! He was {abs(total_time-word_records):.2f} seconds slower.")